### PR TITLE
Feature/us10

### PR DESCRIPTION
**Pull request by Liam-Brew**
Added check for marriage occurring after the age of 14

#14

### CODE REVIEW

File: project/validators/MarriageAfter14.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            Family family = entry.getValue();
+
+            Calendar calendar = Calendar.getInstance();
+            calendar.setTime(family.getMarriage());
+
+            int marriage = calendar.get(Calendar.YEAR);
+
+            Individual husband = individuals.get(family.getHusband());
+            calendar.setTime(husband.getBirthday());
+            int husbandBirth = calendar.get(Calendar.YEAR);
```
Comment (aej11a) [inline project/validators/MarriageAfter14.java:31]: Will this check exact birthdates or only check based on the year of birth?

Comment (Liam-Brew) [reply project/validators/MarriageAfter14.java]: Good catch, I updated to check the full date by comparing birthday+14 years to marriage

**Comment by Liam-Brew** [comment]
Added unit testing

**Review by robertschaedler3 — APPROVED** [review]
Great testing coverage!

**Review by aej11a — APPROVED** [review]

**Comment by aej11a** [comment]
For some reason, I can't fix the conflicts in github. Can you pull and check it out when you have time @Liam-Brew ?

**Comment by aej11a** [comment]
@Liam-Brew Looks like there's conflicts in the xlsx file, it might be best to just remove that file from this commit. I think we'll update the xlsx file together during our sprint review.

**Comment by Liam-Brew** [comment]
Deleted the .xlsx file, but merge request is still blocked. Ill look into it more tomorrow

**Pull request merged**